viewmodel-api/src/main/java/org/cytoscape/view/model/CyColumnView.java

```diff
@@ -0,0 +1,38 @@
+package org.cytoscape.view.model;
```

Comment:
This file is probably one of the most important parts of the design. It allows a VisualProperty to be set on a column but have a different value for each cell in that column. I think the renderer itself needs to control when a mapping is evaluated for performance reasons. This interface allows the view model, which is directly accessible to the renderer, to have a direct reference to the mapping.

Reply:
I'm confused by this.  Not clear why we would want a different property value for each row.  In this case, the analogy is a standard node visual property -- we don't need this for nodes, why would we need it for rows?

Reply:
Not sure why this is necessary in the API, or at least it should be flagged as a byproduct of the implementation.